src/components/Board/Board.component.js

```diff
@@ -381,29 +381,29 @@ export class Board extends Component {
             'is-locked': this.props.isLocked
           })}
         >
-          {isLocked && isTourEnabled && (
+          {isLocked && isRootBoardTourEnabled && (
             <Joyride
               callback={() => {}}
               steps={this.lockedHelpSteps}
               continuous={true}
               showSkipButton={true}
               showProgress={true}
               disableOverlayClose={true}
-              run={isTourEnabled}
+              run={isRootBoardTourEnabled}
               styles={joyRideStyles}
               locale={{
                 last: <FormattedMessage {...messages.walkthroughEndTour} />,
                 skip: <FormattedMessage {...messages.walkthroughCloseTour} />
               }}
             />
           )}
-          {!isLocked && isTourEnabled && (
+          {!isLocked && isRootBoardTourEnabled && (
```

Comment:
since now we have a state for every tour, this element should use a different state than `isRootBoardTourEnabled`. You can use the name `isUnlockedTourEnabled`